core/src/main/java/org/apache/accumulo/core/metadata/schema/AmpleImpl.java

```diff
@@ -44,7 +44,7 @@ public TabletMetadata readTablet(KeyExtent extent, ReadConsistency readConsisten
     builder.readConsistency(readConsistency);
 
     try (TabletsMetadata tablets = builder.build()) {
-      return Iterables.getOnlyElement(tablets);
+      return tablets.stream().collect(onlyElement());
```

Comment:
do we need `onClose(tablets::close)` here to close the underlying Scanner?

Reply:
Nevermind, the stream is not being returned, it's being collected and the collection is returned.